exonum-java-binding/core/rust/exonum-java/src/node.rs

```diff
@@ -43,57 +44,50 @@ pub fn run_node(command: Command) -> Result<(), failure::Error> {
 
 fn create_node(config: Config) -> Result<Node, failure::Error> {
     let node_config = config.run_config.node_config.clone();
-    let events_pool_capacity = &node_config.mempool.events_pool_capacity;
+    let events_pool_capacity = &node_config.private_config.mempool.events_pool_capacity;
     let channel = NodeChannel::new(events_pool_capacity);
     let blockchain = create_blockchain(&config, &channel)?;
 
-    let node_config_path = config
-        .run_config
-        .node_config_path
-        .to_str()
-        .expect("Cannot convert node_config_path to String")
-        .to_owned();
-
     Ok(Node::with_blockchain(
         blockchain,
         channel,
-        node_config,
-        Some(node_config_path),
+        node_config.into(),
+        // TODO: use DefaultConfigManager once it is available
```

Comment:
Fix have already been submitted to the Core.

Reply:
What does that imply for this PR? Shall we wait till it is integrated?

Reply:
Fixed now, updated to revision b81d64ef which includes the fix